packages/asset-swapper/src/types.ts

```diff
@@ -190,6 +190,8 @@ export interface SwapQuoteBase {
     isTwoHop: boolean;
     makerTokenDecimals: number;
     takerTokenDecimals: number;
+    ethToInputRate: BigNumber;
```

Comment:
can we rename them as `takerAssetDataToEthRate` and `makerAssetDataToEthRate` (or eth to x) ? `Input` and `output` are confusing IMO